src/components/menu/menu.tsx

```diff
@@ -90,6 +90,13 @@ export class Menu {
         this.portalId = createRandomString();
     }
 
+    public componentWillLoad() {
+        /* eslint-disable-next-line no-console */
+        console.warn(
+            "The `iconColor` prop is deprecated now! Use the new `Icon` interface and instead of `iconColor: 'color-name'` write `icon {name: 'icon-name', color: 'color-name'}`."
+        );
+    }
```

Comment:
We could warn only if the `iconColor` attribute is used.

Reply:
oh yes please! how to do that. give me an example and I'll fix it everywhere

Reply:
I'm not sure if the input values are available yet in `componentWillLoad`, so you might have to do it in `componentDidLoad` instead, but you could just look at `this.iconColor`. If it has a truthy value, log the warning.

Reply:
I'll test